src/main/java/io/jenkins/plugins/levo/helpers/LevoDockerTool.java

```diff
@@ -41,12 +41,41 @@ public class LevoDockerTool {
     public static final int CLIENT_TIMEOUT = 1800;
     public static final String ENV_FILE_NAME = "environment.yaml";
     public static final String LEVO_CONFIG_FOLDER_NAME = ".levoconfig";
+    public static final String LEVO_REPORTS_FOLDER_NAME = "levo-reports";
+
+    private static String getUserId(Launcher launcher, EnvVars launchEnv) throws IOException, InterruptedException {
+        Launcher.ProcStarter procStarter = launcher.launch();
+        ByteArrayOutputStream baos = new ByteArrayOutputStream();
+        ArgumentListBuilder argb = new ArgumentListBuilder();
+        argb.add("id", "-u");
+        procStarter.quiet(false)
+                .cmds(argb)
+                .envs(launchEnv)
+                .stdout(baos)
+                .start()
+                .joinWithTimeout(CLIENT_TIMEOUT, TimeUnit.SECONDS, launcher.getListener());
+        return baos.toString(StandardCharsets.UTF_8.name()).trim();
+    }
+
+    private static String getUserGroupId(Launcher launcher, EnvVars launchEnv) throws IOException, InterruptedException {
```

Comment:
how about pulling the common code to run command into one method and just passing the command a args?